appengine/standard/endpoints-frameworks-v2/backend/app.yaml

```diff
@@ -10,18 +10,11 @@ handlers:
 libraries:
 - name: pycrypto
   version: 2.6
-
-# Endpoints Frameworks 2.0 uses background threads for caching and reporting to
-# service management and service control.
-manual_scaling:
-  instances: 1
-
-beta_settings:
-  use_endpoints_api_management: true
-  endpoints_swagger_spec_file: echo-v1_swagger.json
+- name: ssl
+  version: latest
```

Comment:
Please specify the version, in this case, '2.7.11'

(although ssl will soon be enabled by default)